cypress/support/commands/navigation.js

```diff
@@ -119,3 +120,29 @@ Cypress.Commands.add('expectNumberOfRowsToBeGreaterThan', numberOfRows => {
     expect(el).to.have.length.greaterThan(numberOfRows);
   });
 });
+
+Cypress.Commands.add('selectRowByColumnAndValue', (columnName, expectedValue, modal = false, force = false) => {
+  cy.log(`Select row by columnName=${columnName} and expectedValue=${expectedValue}`);
+  if (!force) {
+    cy.waitForSaveIndicator();
+  }
+  let path = '.table-flex-wrapper';
+
+  if (modal) {
+    path = '.modal-content-wrapper ' + path;
+  }
+
+  // step 1: figure out from the thead what is the column index
+  return cy
+    .get(path)
+    .contains('thead tr th', columnName)
+    .invoke('index')
+    .then(columnIndex => {
+      // step 2: return the cell which contains the particular soDocNumber in columnNumber from above
+      return cy
```

Comment:
soDocNumber meaning sales order doc number?

Reply:
Fixed in my other branch: https://github.com/metasfresh/metasfresh-e2e/issues/292